client/jetpack-connect/plans.jsx

```diff
@@ -96,6 +97,10 @@ class Plans extends Component {
 	redirectToCalypso() {
 		const { canPurchasePlans, selectedSiteSlug } = this.props;
 
+		if ( selectedSiteSlug && canPurchasePlans && config.isEnabled( 'jetpack/checklist' ) ) {
+			return this.redirect( CALYPSO_MY_PLAN_PAGE );
+		}
+
 		if ( selectedSiteSlug && canPurchasePlans ) {
 			// Redirect to "My Plan" page with the "Jetpack Basic Tour" guided tour enabled.
 			// For more details about guided tours, see layout/guided-tours/README.md
```

Comment:
@johnHackworth  Is our plan to get rid of `{ tour: 'jetpack' }` and replace it with the checklist?

Reply:
@simison @johnHackworth Is the "tour" you're talking about the super brief guided tour that's displayed in calypso after a jetpack plan purchase?

Reply:
> Is the "tour" you're talking about the super brief guided tour that's displayed in calypso after a jetpack plan purchase?

That's the tour, yes.